db/seeds/base/add_declarations.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+LeadProvider.find_each do |lead_provider|
+  quantity = { "review" => 4, "development" => 1 }.fetch(Rails.env, 0)
+
+  quantity.times do
+    # Application with a started declaration
+    application1 = FactoryBot.create(
+      :application,
+      :accepted,
+      :with_random_user,
+      :with_random_work_setting,
+      lead_provider:,
+      course: Course.all.sample,
+      cohort: Cohort.all.sample,
+    )
+    %w[started].each do |declaration_type|
```

Comment:
do we wanna check eligibility to mark ensure its marked as eligible or submitted? or are they all submitted?